react/client/public/locales/en/translation.json

```diff
@@ -0,0 +1,13 @@
+{
+  "disclaimerText": {
+    "0": "Please make sure you are eligible for getting your record cleared. If you need help with this please email placeholder@website.com.",
+    "1": "Be aware your statement is stored locally on your device, and never transmitted to our servers or any third-party",
+    "2": "This software is developed by a non-governmental third party, Code for America.",
+    "3": "Please also review our Privacy Policy and Terms of Use before beginning.",
+    "4": "Please allocate at least 30 minutes of time to complete this.",
+    "5": "If at anytime you are confused please click the question mark button for guidance."
+  },
+  "disclaimerHeader": {
+    "0": "Before you begin"
+  }
```

Comment:
I get why you had to use numbers for access keys on the above one but ideally we can be descriptive with our keys - maybe this one could be `disclaimer.headerText`

Reply:
Great idea, done